load-test-framework/src/main/proto/command.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+package pubsub;
+
+option java_package = "com.google.pubsub.flic.common";
+option java_outer_classname = "Command";
+
+service LoadtestFramework {
+  rpc StartClient(CommandRequest) returns (google.protobuf.Empty) {
```

Comment:
Does "command" have any special meaning? Why is the file called command.proto? Generally, the name of the request message should be <method_name>Request. So here it would StartClientRequest. And the response should be StartClientResponse -- it can be empty for now, but we may want to add fields to it later.